tests/package/test_package.py

```diff
@@ -1,17 +1,26 @@
 import siliconcompiler
-from siliconcompiler import dependency
+from siliconcompiler import package
 from pathlib import Path
 import pytest
 import logging
+import os
 
 
-def dependency_cache_path(path, ref, chip=None, quiet=True):
+@pytest.fixture(autouse=True)
+def mock_home(monkeypatch):
```

Comment:
What were your thoughts for adding this?

Reply:
I prefer the tests not modify my local setup and run in the pytest directory

Reply:
Sounds reasonable. Should we do that globally then?
Any test that uses a target will download or use the cache as well.

Reply:
Good point, can you handle that? just push the changes to this PR